spec/features/feature_flags_spec.rb

```diff
@@ -46,4 +46,58 @@
       end
     end
   end
+
+  describe 'search and sort' do
+    let!(:myuser) { create :user }
+
+    context 'disabled' do
+      before do
+        ENV['SEARCH_AND_SORT'] = 'false'
+        login_as(myuser, :scope => :user)
+        visit clients_path
+      end
```

Comment:
This needs an `after` block to set the SEARCH_AND_SORT value to be whatever it was previously. See https://github.com/codeforamerica/clientcomm-rails/blob/53635be15655e5f9938b3dbec5cb98b644a4670c/spec/features/feature_flags_spec.rb#L6-L12